.github/workflows/snap.yaml

```diff
@@ -27,27 +25,27 @@ jobs:
         name: Checkout repo
         uses: actions/checkout@v3
         with:
-          ref: dev
+          ref: ${{ env.BRANCH }}
       - id: published-version
-        name: Read last published stable version
+        name: Read last published stable Spark version
         uses: juliangruber/read-file-action@v1
         with:
-          path: ./VERSION
+          path: ./SPARK_VERSION
       - id: latest-version
         name: Check latest stable version available for download
         run: |
           LATEST_STABLE_SPARK_VERSION=$(curl --silent https://downloads.apache.org/spark/ | grep "spark-" | cut -d'>' -f3 | cut -d'/' -f1  | sort | tail -n 1)
```

Comment:
Can you please make two different variables one for the full version name `spark-3.3.1` and another one with the version only `3.3.1` ?